sdk/tables/azure-data-tables/test/ut/table_client_test.cpp

```diff
@@ -469,6 +469,41 @@ namespace Azure { namespace Data { namespace Test {
     EXPECT_EQ(responseQuery.TableEntities.size(), 1);
   }
 
+  TEST_P(TablesClientTest, EntityGet)
+  {
+    if (GetParam() == AuthType::Key)
+    {
+      EXPECT_TRUE(true);
+      return;
+    }
```

Comment:
Why do we have this type of parameterized test if it doesn't make sense here? Deserves a comment.